templates/policy-compliance-agent/backend/services/vector_db_manager.py

```
@@ -0,0 +1,389 @@
+                query_texts=[query],
+                n_results=n_results,
+                include=["documents", "metadatas", "distances"],
+            )
+
+            search_results = []
+            for i, (document, metadata, distance) in enumerate(
+                zip(
+                    results["documents"][0],
+                    results["metadatas"][0],
+                    results["distances"][0],
+                )
```

Ruff (0.14.5)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)